.github/workflows/fork-pr-notice.yml

```diff
@@ -25,7 +25,7 @@ jobs:
               body: `Thank you for your contribution! Here are a few things to check in the PR to ensure it's reviewed as quickly as possible:
             
               - Your branch should be based on \`origin/develop\`, at least when it was created.
-              - There is a changelog entry in the \`changelog.d\` folder with [the Towncrier format](https://towncrier.readthedocs.io/en/latest/tutorial.html#creating-news-fragments).
+              - The title of the PR will be used for release notes, so it need to describe the change visible to the user.
```

Comment:
Nit: `it needs`

Reply:
https://github.com/element-hq/element-x-android/pull/3105/commits/790c0aa0c0dc8399e8f6691f3b2fe725097a99b4